### PR TITLE
Write minValidatorCount to genesis.json

### PR DESCRIPTION
**Pull request by jp-imx**
Write the value from the genesis '--min-validator-count' flag to the genesis.json output

# Description

Please provide a detailed description of what was done in this PR

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [ ] I have assigned this PR to myself
- [ ] I have added at least 1 reviewer
- [ ] I have added the relevant labels
- [ ] I have updated the official documentation
- [ ] I have added sufficient documentation in code

## Testing

- [ ] I have tested this code with the official test suite
- [x] I have tested this code manually

### Manual tests

I ran the following command and verified the file:

```
./polygon-edge genesis --block-gas-limit 10000000 --epoch-size 10 \                                     
--validators-path ./ --validators-prefix test-chain- \
--consensus polybft \
--reward-wallet ... \
--transactions-allow-list-admin ... \
--transactions-allow-list-enabled ... \
--block-gas-limit ... \
--block-time ... \
--min-validator-count 3 \
--max-validator-count 100
```

# Documentation update

Please link the documentation update PR in this section if it's present, otherwise delete it

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

File: consensus/polybft/polybft_config.go [hunk 1]
```diff
@@ -43,6 +43,9 @@ type PolyBFTConfig struct {
 	// SupernetID indicates ID of given supernet generated by stake manager contract
 	SupernetID int64 `json:"supernetID"`
 
+	// MaxValidatorSetSize indicates the maximum size of validator set
```
Comment (Stefan-Ethernal) [inline consensus/polybft/polybft_config.go]:
```suggestion
	// MinValidatorSetSize indicates the minimum size of validator set
```

**Review by Stefan-Ethernal — COMMENTED** [review]
Curious how you envision using the `MinValidatorSetSize` parameter. Currently, it is not used for the polybft consensus protocol, so does it make sense to exist in the first place?

**Comment by jp-imx** [comment]
@Stefan-Ethernal, we are doing some prep work to understand the configurable parameters, how they influence the behaviour of our chain, and make decisions on what the genesis for our public testnet and mainnet should look like. While doing that, I found that the `genesis.json` file was not documenting this setting. I have yet to look into how it would apply to our chain, if at all.

Can you explain what the intended use of `MinValidatorSetSize` is ATM?

**Comment by goran-ethernal** [comment]
> @Stefan-Ethernal, we are doing some prep work to understand the configurable parameters, how they influence the behaviour of our chain, and make decisions on what the genesis for our public testnet and mainnet should look like. While doing that, I found that the `genesis.json` file was not documenting this setting. I have yet to look into how it would apply to our chain, if at all.
> 
> Can you explain what the intended use of `MinValidatorSetSize` is ATM?

We will definitely use it in on-chain governance, since we are starting to implement that feature for the next release. We will be using it to update the current active validator set. We also added it to `PolybftConfig` in that governance PR we started: https://github.com/0xPolygon/polygon-edge/pull/1519/files#diff-8291393e7671084ce64a724f76f24df16cf0e262de84905168c4667f94a320abR43

**Comment by jp-imx** [comment]
@goran-ethernal seems like the PR you referenced already includes the changes in the branch. I will close this PR as redundant.

**Comment by goran-ethernal** [comment]
> @goran-ethernal seems like the PR you referenced already includes the changes in the branch. I will close this PR as redundant.

Nah, we can merge your PR ASAP. No need to wait for ours.

**Pull request closed without merging**